HeterogeneousCore/CUDACore/interface/CUDAESProduct.h

```diff
@@ -86,16 +86,18 @@ class CUDAESProduct {
 
     return data.m_data;
   }
-  
+
 private:
   struct Item {
     mutable std::mutex m_mutex;
     CMS_THREAD_GUARD(m_mutex) mutable std::unique_ptr<cuda::event_t> m_event;
-    CMS_THREAD_GUARD(m_mutex) mutable cudaStream_t m_fillingStream = nullptr; // non-null if some thread is already filling (cudaStream_t is just a pointer)
-    mutable std::atomic<bool> m_filled = false; // easy check if data has been filled already or not
+    CMS_THREAD_GUARD(m_mutex)
```

Comment:
this is not nice either, especially since the line above has the same length ans is unchanged ...

Reply:
Can be likely fixed by moving the comments.